File: Data Structures & Algorithms/remove-element/submission-0.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int last_ind = 0;
        for(int i =0;i<nums.size();++i) {
            if (nums[i] != val) {
                nums[last_ind] = nums[i];
                ++last_ind;
            }
        }
        return last_ind;
    }
    
};

File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        mx_reach = 0
        for i in range(len(nums)):
            if i > mx_reach:
                return False
            mx_reach = max(mx_reach,nums[i] + i)
            if mx_reach >= len(nums)-1:
                return True






